tools/generate_grpc_docs.dockerfile

```
@@ -0,0 +1,19 @@
+FROM pseudomuto/protoc-gen-doc:latest
+
+WORKDIR /protos
+
+ADD lib/api/src/grpc/proto/ .
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)